tests/{{cookiecutter.repo_name}}.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -ex
+
+source $BASE_DIR/admin-openrc
+
```

Comment:
Maybe worth adding a comment at the end of this file to say something like "here's where you put your setup and tests"